pulp_rpm/app/depsolving.py

```diff
@@ -409,7 +409,7 @@ def dep_or_rel(dep, op, rel):
           dep_or_rel(solv.Dep, solve.REL_WITH, stream_dep(name, stream))
           dep_or_rel(solv.Dep, solve.REL_WITHOUT, stream_dep(name, stream))
         """
-        dep.Rel(op, rel) if dep is not None else rel
+        return dep.Rel(op, rel) if dep is not None else rel
```

Comment:
oof.

Reply:
oops!

Reply:
these is my favorite type of bug chasing >,>

Reply:
and wrong indentation one, obviously

Reply:
@ipanova Do you want to add a review?